core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -419,7 +427,32 @@ public void ddl(CharSequence ddl, SqlExecutionContext sqlExecutionContext, @Null
         }
     }
 
-    public void drop(Path path, TableToken tableToken) {
+    public void drop(CharSequence dropSql, SqlExecutionContext sqlExecutionContext) throws SqlException {
+        drop(dropSql, sqlExecutionContext, null);
+    }
+
+    public void drop(CharSequence dropSql, SqlExecutionContext sqlExecutionContext, @Nullable SCSequence eventSubSeq) throws SqlException {
+        try (SqlCompiler compiler = getSqlCompiler()) {
+            final CompiledQuery cq = compiler.compile(dropSql, sqlExecutionContext);
+            switch (cq.getType()) {
+                case UPDATE:
```

Comment:
UPDATE fall-through to DROP? If this is intentional then it must have a comment.

Reply:
this looks sus, removing UPDATE + running tests

Reply:
removed UPDATE in https://github.com/questdb/questdb/pull/5159/commits/c44db28a716a873faaa4c76d7eaad2c55359aacc